### PR TITLE
Design improvements

### PR DESCRIPTION
**Pull request by rosadiaz**

### CODE REVIEW

File: rails-snow-queen/app/assets/javascripts/quote.js [hunk 1]
```diff
@@ -21,8 +21,17 @@ class Quote {
...
+    const primaryAddressNode = document.getElementById("primaryAddressModal");
+    const secondaryAddressNode = document.getElementById("secondaryAddressModal");
+    splitAddress.forEach((element, i) => {
+      const div = document.createElement("div");
+      div.innerText = element;
+      if (i === 0) { 
+        primaryAddressNode.appendChild(div) 
+      } else {
+        secondaryAddressNode.appendChild(div);
+      }
```
Comment (diazruy) [inline rails-snow-queen/app/assets/javascripts/quote.js]: [`Array.prototype.shift`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift)
```javascript
const primaryAddressNode = document.
  getElementById('primaryAddressModal').
  appendChild(splitAddress.shift());
const secondaryAddressNode = document.getElementById("secondaryAddressModal");
splitAddress.forEach((element, i) => {
  const div = document.createElement("div");
  div.innerText = element;
  secondaryAddressNode.appendChild(div);
});
```

File: rails-snow-queen/app/assets/javascripts/quotingPanel.js [hunk 2]
```diff
@@ -14,16 +14,21 @@ class QuotingPanel {
...
     if (geocodedAddress) {
       this.geocodedAddress = geocodedAddress;
-      const addressNode = document.getElementById("displayAddress");
+      const primaryAddressNode = document.getElementById("primaryAddress");
+      const secondaryAddressNode = document.getElementById("secondaryAddress");
       let splitAddress = geocodedAddress.split(",");
-      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
-      splitAddress.forEach(element => {
+      while (primaryAddressNode.firstChild) { primaryAddressNode.removeChild(primaryAddressNode.firstChild) }
+      while (secondaryAddressNode.firstChild) { secondaryAddressNode.removeChild(secondaryAddressNode.firstChild) }
```
Comment (diazruy) [inline rails-snow-queen/app/assets/javascripts/quotingPanel.js:20]: Maybe add a method for this: `emptyNode(node)`. Seems like you use it more than once.

File: rails-snow-queen/app/assets/javascripts/quotingPanel.js [hunk 3]
```diff
@@ -14,16 +14,21 @@ class QuotingPanel {
...
+      splitAddress.forEach((element, i) => {
         const div = document.createElement("div");
         div.innerText = element;
-        addressNode.appendChild(div);
+        if (i === 0) {
```
Comment (diazruy) [inline rails-snow-queen/app/assets/javascripts/quotingPanel.js]: See comment on https://github.com/rosadiaz/snow_queen_v2/pull/9/files#diff-9067261253110cf18cd55210cbc898f8R33

File: rails-snow-queen/app/assets/javascripts/quotingPanel.js [hunk 4]
```diff
@@ -14,16 +14,21 @@ class QuotingPanel {
...
-        addressNode.appendChild(div);
+        if (i === 0) { 
+          primaryAddressNode.appendChild(div) 
+        } else {
+          secondaryAddressNode.appendChild(div);
+        }
       });
-      addressNode.classList.remove("hidden");
-      document.getElementById("searchAddressButton").classList.remove("hidden");
+      document.getElementById("displayAddress").classList.remove("hidden");
```
Comment (diazruy) [inline rails-snow-queen/app/assets/javascripts/quotingPanel.js:31]: Maybe add a method for `showNode(id)` (used in several places)

File: rails-snow-queen/app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -12,21 +12,31 @@
...
+  "warning": #D6C9C9,
+  "secondary": gray,
+  "info": #F6F7F7,
   "dark": #177E89
 );
 
 @import 'bootstrap';
 
+body {
+}
```
Comment (diazruy) [inline rails-snow-queen/app/assets/stylesheets/application.scss]: Cruft?

File: rails-snow-queen/app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -12,21 +12,31 @@
  *
  */
 $theme-colors: (
-  "primary":#61D5DD,
-  "warning":#fa7f28,
+  "light":#ddf8fa,
+  "primary":#0698b190,
```
Comment (diazruy) [inline rails-snow-queen/app/assets/stylesheets/application.scss]: Consistent spacing after `:`

File: rails-snow-queen/app/assets/stylesheets/application.scss [hunk 7]
```diff
@@ -36,7 +46,6 @@ header {
 }
 
 #panel {
-  background-color: #177E89;
```
Comment (diazruy) [inline rails-snow-queen/app/assets/stylesheets/application.scss:39]: Can remove the selector now if no styles

File: rails-snow-queen/app/assets/stylesheets/application.scss [hunk 8]
```diff
@@ -65,4 +78,23 @@ header {
...
+  color: gray;
+}
+
+.font-size-35 {
+  font-size: 35px;
+}
+
+.currency-symbol {
+  font-size: 25px;
+  vertical-align: super;
```
Comment (diazruy) [inline rails-snow-queen/app/assets/stylesheets/application.scss:93]: How come this needs special alignment?

**Review by diazruy — COMMENTED** [review]
Almost there!

File: rails-snow-queen/app/assets/javascripts/quote.js (hunk 1 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/javascripts/quote.js]: done

File: rails-snow-queen/app/assets/javascripts/quotingPanel.js (hunk 3 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/javascripts/quotingPanel.js]: done

File: rails-snow-queen/app/assets/stylesheets/application.scss (hunk 5 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/stylesheets/application.scss]: done

File: rails-snow-queen/app/assets/stylesheets/application.scss (hunk 6 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/stylesheets/application.scss]: done

File: rails-snow-queen/app/assets/stylesheets/application.scss (hunk 7 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/stylesheets/application.scss:39]: done

File: rails-snow-queen/app/assets/stylesheets/application.scss (hunk 8 above)
Comment (rosadiaz) [reply rails-snow-queen/app/assets/stylesheets/application.scss:93]: to format the '$' independent from the amount

Comment (diazruy) [reply rails-snow-queen/app/assets/stylesheets/application.scss:93]: I was referring to the `vertical-align` specifically. What's the deal with `super`?

**Pull request merged**